src/turboServer.ts

```diff
@@ -13,7 +13,7 @@ async function startServer() {
   fs.ensureDirSync(cacheDir);
 
   const app = express();
-  const serverToken = getInput(Inputs.SERVER_TOKEN, {
+  const serverToken = process.env.TURBO_TOKEN || getInput(Inputs.SERVER_TOKEN, {
```

Comment:
I left the `server-token` as an input, for backwards compatibility. So, it won't break if someone doesn't update their workflow.

Reply:
How do you pass `TURBO_TOKEN` to the local server as environment variable?
I'm got confused, since I'm not using this approach, it is not clear to me how it get's it during the start...

In the README, you've changed the fact that it is required input.